pkg/service/system/route53.go

```diff
@@ -40,6 +43,8 @@ var regionClassicLoadBalancerHostedID = map[string]string{
 	"us-gov-west-1":  "Z33AYJ8TM3BH4J",
 }
 
+const defaultRegion = "us-west-1"
```

Comment:
Don't like the idea of using a default region here, can we not get it from the request?